### PR TITLE
std.math: №4, workaround for intrinsics

### PR DESCRIPTION
**Pull request by 9il**
~~Depends on https://github.com/D-Programming-Language/dmd/pull/4425~~

### CODE REVIEW

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
@9il do you have anything to add here?

**Comment by 9il** [comment]
No, I have not. Other stuff like `log` can be implemented in D.

**Comment by burner** [comment]
Auto-merge toggled on

**Comment by 9il** [comment]
Thanks!

**Comment by burner** [comment]
thank you

File: std/math.d [hunk 1]
```diff
@@ -1384,6 +1396,12 @@ unittest
  * indeterminate.
  */
 long rndtol(real x) @nogc @safe pure nothrow;    /* intrinsic */
+//FIXME
+///ditto
+double rndtol(double x) @safe pure nothrow @nogc { return rndtol(cast(real)x); }
+//FIXME
+///ditto
+float rndtol(float x) @safe pure nothrow @nogc { return rndtol(cast(real)x); }
```
Comment (don-clugston-sociomantic) [inline std/math.d]: ??? Why did this get merged? The return type is obviously wrong!

Comment (9il) [inline std/math.d]: O_o. I will create new PR right now

Comment (9il) [inline std/math.d]: Thanks! PR #3166

**Pull request merged**